src/main/java/frc/robot/subsystems/Pneumatics.java

```diff
@@ -0,0 +1,88 @@
+// Copyright (c) FIRST and other WPILib contributors.
```

Comment:
Not important but style suggestion: be consistent about whether you use period in Javadocs or not

Reply:
I don't usually